Doc/library/xml.etree.elementtree.rst

```diff
@@ -1057,6 +1057,11 @@ Element Objects
      if element is None:
          print("element not found")
 
+   .. versionchanged:: 3.12
+      Prior to Python 3.12, this usage was discouraged by documentation, and
```

Comment:
Lets leave the messy history out here and just focus on the meaningful visible change: this now always raises a DeprecationWarning.